### PR TITLE
div with classname appears when profiles fetched

### PR DESCRIPTION
**Pull request by ShovelHand**
Hi, Billy. 
I think this is what you had in mind, but one thing I'm not sure about is how searching for elements by className in React works for external stuff, like entity. 
Let me know if you need me to rework this.

### CODE REVIEW

**Comment by ShovelHand** [comment]
duplicate of 794

File: web/src/views/Dashboard.tsx [hunk 1]
```diff
@@ -16,6 +16,7 @@
 
 import { useKeycloak } from '@react-keycloak/web';
 import React, { useEffect, useState } from 'react';
+import { Bool } from 'reselect/es/types';
```
Comment (w8896699) [inline web/src/views/Dashboard.tsx:19]: You are not using this Bool why add here?

**Review by w8896699 — CHANGES_REQUESTED** [review]
Almost there

File: web/src/views/Dashboard.tsx (hunk 1 above)
Comment (ShovelHand) [reply web/src/views/Dashboard.tsx]: Woops. Removed that.

**Pull request merged**